<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevInfoSys</class>
 <widget class="QDialog" name="DevInfoSys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册设备信息</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>181</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="iDLabel">
      <property name="text">
       <string>设备ID：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="iDLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="devNameLabel">
      <property name="text">
       <string>设备名称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="devNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="deviPLabel">
      <property name="text">
       <string>设备IP：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="deviPLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="fTPIPLabel">
      <property name="text">
       <string>FTP服务器IP：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="fTPIPLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>191</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="fTPLabel">
      <property name="text">
       <string>FTP抓拍存放路径：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="fTPCaptureLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="fTPLabel_2">
      <property name="text">
       <string>FTP识别存储路径：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="fTPRecogLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>实时事件端口：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="rtcPortLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>主机报警端口</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="warnPortLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="isCannecl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
